fetcher/fetcher.go

```diff
@@ -36,8 +37,8 @@ func (fetcher baseFetcher) GetMonthly() *prometheus.GaugeVec {
 	return fetcher.monthly
 }
 
-func newBase(pricing *PriceProvider, resource string, additionalLabels ...string) *baseFetcher {
-	labels := []string{"name"}
+func newBase(pricing *PriceProvider, resource string, baselabels []string, additionalLabels ...string) *baseFetcher {
```

Comment:
I don't think it is necessary to differantiate between the `baseLabels` and the `additionalLabels` now, is it? The parameter could just be `labels ...string`, which would make all call sites alot cleaner, imo.

Its just a small refactoring, which I can also do if you want to get done with this PR, your call 🙂

Reply:
It bredd to be differentiated in the way they are fetched from the object. The baseLabels are fetched from individual fields of the object.

The additional labels are fetched from the metadata in the labels map, using the mapping function.

Reply:
I am stupid. You are so right 😄